docs/cristin.ttl

```diff
@@ -72,6 +72,37 @@
     and they have contributed as "ILLUSTRATOR" affiliated with InstB for the same publication.
     """ .
 
+:TYPE_MEDIEBIDRAG_Class
+    a            :CristinClass ;
+    rdfs:comment """
```

Comment:
har følgende kategorier:
DOKUMENTAR
INTERVJU
PROGDELTAGELSE
PROGLEDELSE

i ett av følgende medium:
AVIS
TV
INTERNETT
RADIO
TIDSSKRIFT
FAGBLAD

Reply:
In the export we receive the following object:
 "type_mediebidrag": {
    "varbeidlopenr": 965431
  }
and it seems that it is always equal to respective Cristin result ID, i.e., it is always  a self reference and therefore we can ignore it. Is my assumption correct?